### PR TITLE
Retry on Linode Busy errors.

### PR DESCRIPTION
**Pull request by phillc**

### CODE REVIEW

**Comment by displague** [comment]
@phillc you may be interested in comparing notes with https://github.com/linode/linodego/pull/70/files.  The fixture in that PR was hand-crafted because generating this fixture dynamically was not guaranteed to be reproducible.

File: client.go [hunk 1]
```diff
@@ -154,10 +154,30 @@ func (c *Client) SetToken(token string) *Client {
 	return c
 }
 
+func (c *Client) SetLinodeBusyRetry() *Client {
```
Comment (displague) [inline client.go]: drive-by comments..


`Linode` may be redundant naming here.  OTOH, `SetAPI*` is patterned above.

A type that defines the retry values could be useful, and this function could take a reference to a variable of that type.

This function should have  a go doc and it should call out the behavior  `SetPollDelay` has on this.

Comment (phillc) [reply client.go]: Point 1, I would have named the method "Server Busy" if the error message said that. Its the only error I am targeting right now.

I understand your point on the second. I will keep this in mind for refactor in the future.

Addressed point 3
